src/dataregistry/query.py

```diff
@@ -48,6 +48,32 @@
 def is_orderable_type(ctype):
     return type(ctype) in ALL_ORDERABLE
 
+def _convert_alchemy_to_dict(results):
+    """
+    Function to convert a sqlalchemy.engine.Result object to a dict.
+
+    Parameters
+    ----------
+    result : sqlalchemy.engine.Result object
+
+    Returns
+    -------
+    data : dict
+        Each row in the results table becomes a key
```

Comment:
I don't think this is what you mean.    The keys are the column names of the requested columns; the value associated with a key is a list of the values for that column The **rows** from the results table don't become anything; they're split up.